ast/dml.go

```diff
@@ -187,12 +187,15 @@ type TableName struct {
 }
 
 // Restore implements Node interface.
-func (n *TableName) Restore(ctx *RestoreCtx) error {
+func (n *TableName) restoreName(ctx *RestoreCtx) {
```

Comment:
why the permission of this file has been changed?

Reply:
This file, with go source code, currently has "execute" permission set. This PR removes "execute" bit.
it was set by 02812c3c22e70aa0acd643918ac2152418c11e7c commit on Jan 8 12:41:00 2019

that commit (02812c3c22e70aa0acd643918ac2152418c11e7c) set executable flag on may files, including `parser_test.go`.
Later, on  Aug 14 09:47:16 2019 (commit 91c196d9fac2c5b7769b7be6a8e84aa0bc31036c), executable bit was removed from `parser_test.go`

Looks to me like an unintentional change leaked thru by 02812c3c22e70aa0acd643918ac2152418c11e7c.
Does `ast/dml.go` needs to be executable?

Reply:
ok, "644" is fine.